step2-01/README.md

```diff
@@ -122,51 +151,53 @@ async function someFunctionAsync() {
 }
 ```
 
-All functions that are marked `async` return a `Promise` automatically. This previous example returned a `Promise<string>`, and can be used like this:
+All functions that are marked `async` return a `Promise` automatically. The previous example returned a `Promise<string>`, and can be used like this:
 
 ```ts
 someFunctionAsync().then(result => {
   console.log(result);
 });
 ```
 
+> For more information, see [this article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function).
+
 # Exercise
 
-Please complete all exercises inside the `exercise/src` folder unless otherwise specified in the exercises below. First, open up [Step2-01 exercise page](http://localhost:8080/step2-01/exercise/) to see the results while you're implementing things.
+Please complete all exercises inside the `exercise/src` folder unless otherwise specified. First, open up [Step2-01 exercise page](http://localhost:8080/step2-01/exercise/) to see the results while you're implementing things.
 
 ## Modules
 
-1. Open up file called `index.ts` in VS Code
+1. Open the file called `index.ts` in VS Code
 
 2. Create another module file called `fibonacci.ts`
 
-3. Inside the file from (step 2), write a function called `fib(n)` that takes in a number and returns a the n-th Fibonacci number - be sure the specify the type of n
+3. Inside the file from (step 2), write a function called `fib(n)` that takes in a number and returns the `n`-th Fibonacci number (be sure the specify the type of `n`).
 
-> HINT: function fib(n: number) { return n <= 1 ? n : fib(n - 1) + fib(n - 2); }
+> HINT: `function fib(n: number) { return n <= 1 ? n : fib(n - 1) + fib(n - 2); }`
 
 4. Export `fib(n)` as a **named export**
 
 5. Export another const variable as a **default export**
 
-6. Inside `index.ts` Import both the modules created in steps (4) and (5) and use the built-in `console.log()` function to log the result of `fib(FibConst)`.
+6. Inside `index.ts`, import both of the modules created in steps (4) and (5) and use the built-in `console.log()` function to log the result of `fib(FibConst)`.
 
-## Types, Interfaces, and Classes
+## Types and Interfaces
 
 Inside `index.ts`:
 
 1. Add a type alias for string union type describing the states of Red-Green-Yellow traffic light: `type TrafficLight = ???`
 
 2. Describe a type of car with an interface: `interface Car { ... }` complete with `wheels`, `color`, `make`, `model`
 
-## Generic
+## Generics
 
-Inside `stack.ts`, create a generic class for a `Stack<T>` complete with a typed `pop()` and `push()` methods
+Inside `stack.ts`, create a generic class for a `Stack<T>` complete with a typed `pop()` and `push()` methods.
```

Comment:
Given the mixed audience, I wonder if some people won't know what a stack is. This tutorial was the clearest I found in a short search--could potentially add a link to it. https://www.hackerearth.com/practice/data-structures/stacks/basics-of-stacks/tutorial/

Reply:
I think I'll ask up front. But if they don't know I'll just show them with an array :)